.github/workflows/publish-and-deploy.yaml

```diff
@@ -49,5 +49,15 @@ jobs:
         known_hosts: ${{ secrets.GRANDSVC_KNOWN_HOSTS }}
         #config: ${{ secrets.CONFIG }} # ssh_config; optional
         if_key_exists: fail # replace / ignore / fail; optional (defaults to fail)
+
+    - name: Setup WireGuard
+      run:  |
+        run: sudo apt install wireguard
+        echo "${{ secrets.WIREGUARD_PRIVATE_KEY }}" > privatekey
```

Comment:
Is this really the correct way to do this? Won't it get echoed into the publicly readable logs?

Reply:
Invited you to a private repo where I tested something like this, I think it gets masked